snapcraft/internal/deprecations.py

```diff
@@ -27,6 +27,7 @@
 _DEPRECATION_MESSAGES = {
     'dn1': "The 'snap' keyword has been replaced by 'prime'.",
     'dn2': "Custom plugins should now be placed in 'snap/plugins'.",
+    'dn3': "Assets in 'setup/gui' should now be placed in to 'snap/gui'.",
```

Comment:
Small nit: I suggest scratching the "to": "Assets in 'setup/gui' should now be placed in 'snap/gui'."